### ISSUE TITLE
Add CAPOCI to clusterctl 

### ISSUE THREAD

**Issue by joekr** [oracle/cluster-api-provider-oci#29]
**What would you like to be added**:
When a user runs `clusterctl config repositories` we should show information about the CAPOCI provider. We will also want to coordinate with #20 to make sure the docs are updated as well.

**Why is this needed**:
This will make it easier for users to find our provider.

**Comment by joekr** [comment]
We are able to submit changes even though our provider is not yet in the kube-sig repo. So I'll be working on getting that ready and have a PR next week.

**Comment by joekr** [comment]
We have landed https://github.com/kubernetes-sigs/cluster-api/pull/6309 but now waiting on the next release. I think it should be 1.1.4

**Comment by joekr** [comment]
Closing as this is a duplicate of https://github.com/oracle/cluster-api-provider-oci/issues/24
